### ISSUE TITLE
Adopt a different account for RelVal input data placement in MSTransferor

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#10253]
**Impact of the new feature**
MSTransferor

**Is your feature request related to a problem? Please describe.**
Yes, we would like to avoid blocking Release Validation workflows in a scenario where central production has exhausted all the `wmcore_transferor` quota at the usual target RelVal sites.

**Describe the solution you'd like**
We would like to separate central production from RelVal input data placement in MSTransferor. This is the first out of 2 issues.
With this issue, we should implement the following:
* contact the CMS Rucio team and create a new account for RelVal input data placement (how about `wmcore_input_relval`? or `wmcore_relval`?)
* define proper quotas for this account
* add a new parameter to the MSTransferor configuration file which will have the rucio relval account
* in MSTransferor, be able to fetch the relval account limits and the current usage

In short, this issue is meant to prepare all the ground in MSTransferor to separate input data placement for RelVal and for central production.

**Describe alternatives you've considered**
Bypass any quota limits and always make RelVal input data placement, regardless. However, people apparently didn't like this approach :)

**Additional context**
none
